db/seed_data/districts.sql

```diff
@@ -0,0 +1,441 @@
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (31, '17', 6, '2015-02-06 00:48:06.415016', '2015-02-06 00:48:06.416166');
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (32, '18', 6, '2015-02-06 00:48:06.421442', '2015-02-06 00:48:06.42334');
+INSERT INTO districts (id, district, state_id, created_at, updated_at) VALUES (35, '20', 6, '2015-02-06 00:48:06.435113', '2015-02-06 00:48:06.436709');
```

Comment:
Actually,  I don't think we  even need the ID field here.  The table is already created and the sequence rules are set.  Wouldn't postgres take care of ID assignment for us automatically?  

Not worth revisiting now, but maybe some other point.